install_config/configuring_authentication.adoc

```diff
@@ -1435,14 +1435,40 @@ characters long, to select AES-128, AES-192, or AES-256.
 ====
 
 [[configuring-user-agent]]
-== User Agent
+== Preventing CLI Version Mismatch With User Agent
 
-Administrators can prevent clients from accessing the API by their `User-Agent`
-header in the new `*userAgentMatching*` configuration setting.
+{product-title} implements a user agent to prevent an application developer's
```

Comment:
Minor nit and I'm not native so I may be wrong, but I'd change "... implements a user agent to prevent an application developer's ..." to something like "... implements a user agent that can be used to prevent an application developer's ...". The user agent can be used to do this, but it's not the only purpose of having a user agent (it is excepted that every client that talks to a given API provides the User-Agent header, so that it appears in the server-side logs, etc). ;)